src/libraries/Microsoft.Extensions.HostFactoryResolver/src/Microsoft.Extensions.HostFactoryResolver.Sources.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.Build.NoTargets">
```

Comment:
Please add a comment to these project files to indicate that these are only wrapper project files for packaging currently.